content/docs/use-cases/dvc.md

```diff
@@ -22,6 +22,9 @@ $ source .venv/bin/activate
 $ pip install -r requirements.txt
```

Comment:
minor: something is wrong with indent?

Reply:
Sorry, what do you mean? I see no problems
<img width="273" alt="image" src="https://user-images.githubusercontent.com/6797716/178495642-d46103fb-964a-4504-b3bd-9aecf23e6c95.png">

Reply:
please disregard, my bad